crates/relayer/src/chain/cosmos.rs

```diff
@@ -872,7 +872,17 @@ impl ChainEndpoint for CosmosSdkChain {
     type Time = TmTime;
     type SigningKeyPair = Secp256k1KeyPair;
 
+    fn id(&self) -> &ChainId {
+        &self.config.id
+    }
+
     fn bootstrap(config: ChainConfig, rt: Arc<TokioRuntime>) -> Result<Self, Error> {
+        #[allow(irrefutable_let_patterns)]
```

Comment:
(Currently, the pattern is irrefutable, but it won't be in the future, and that should be handled now)